### ISSUE TITLE
Log Getting Spammed:

### ISSUE THREAD

**Issue by blaine07** [martabal/docker-immich#1]
Docker Log getting spammed with: 2023-01-12 01:53:03,778 INFO exited: machine-learning (terminated by SIGILL; not expected)
2023-01-12 01:53:04,781 INFO spawned: 'machine-learning' with pid 225
2023-01-12 01:53:05,783 INFO success: machine-learning entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)

On original container its related to certain chip hardware requirement. On original container set that one specific machine learning container can just not be ran.

Is not running machine learning container a choice here? I can't remember what chip requirement is AFV or something; whatever it is my T610 server doesn't have compatible CPU.

SOS

**Comment by blaine07** [comment]
On this AIO container there isn't a means to change to the NON AVX option:

https://immich.app/docs/install/requirements#hardware

**Comment by blaine07** [comment]
Guess eventually I am going to have to move Unraid off this T610...

https://www.dell.com/community/PowerEdge-Hardware-General/AVX-Support-in-Power-Edge-R610/td-p/6106155

**Comment by martabal** [comment]
Yeah, you need to require a CPU with AVX and AVX2 instruction sets. Someone succeed to create a Dockerfile without needing the AVX / AVX2 instruction set. I'm gonna precise it in the README.
I will first create a script to allow to disable the machine-learning process and then I will create a separate Docker with the  [custom machine-learning](https://github.com/immich-app/immich/discussions/300).

**Comment by martabal** [comment]
@blaine07 I added an environment variable to disable the machine-learning program. Add `DISABLE_MACHINE_LEARNING` to your docker-compose / docker-cli and set it to `true`.

**Comment by blaine07** [comment]
> @blaine07 I added an environment variable to disable the machine-learning program. Add `DISABLE_MACHINE_LEARNING` to your docker-compose / docker-cli and set it to `true`.

THANK YOU! SO much! How long will it take for me to be able to update container so that the env will be seen/used?

THANK YOU!

**Comment by martabal** [comment]
@blaine07 no problem :)
I am re-building the image, it will be available in ~1 hour.

**Comment by martabal** [comment]
You'll just have to pull the docker and you'll be good. Once it's done, can you confirm it's working for you ?

**Comment by blaine07** [comment]
> You'll just have to pull the docker and you'll be good. Once it's done, can you confirm it's working for you ?

Here shortly I can. It’s in Unraid so working quite nicely so far; other than this AVX thing.

**Comment by martabal** [comment]
@blaine07 The machine-learning issue is fixed and the update is available, don't forget to add DISABLE_MACHINE_LEARNING=true to your environment variables and you're good to go

**Comment by blaine07** [comment]
I am in Unraid, added the ENV but it appears it's still doing it?
![Screen Shot 2023-01-12 at 7 21 00 PM](https://user-images.githubusercontent.com/63940698/212215212-5a1b020c-7a6a-4351-b285-1b1e75bd94de.png)
Is that possible?

**Comment by blaine07** [comment]
![Screen Shot 2023-01-12 at 7 21 36 PM](https://user-images.githubusercontent.com/63940698/212215278-eff453c5-f453-4179-aada-f2acec9ef9e5.png)

**Comment by martabal** [comment]
Are you sure to have the last version ?

**Comment by martabal** [comment]
You are supposed to see `machine-learning disabled` when you start the container

**Comment by martabal** [comment]
Access the docker console and execute this command : `ps aux`
You are supposed to see this 
![image](https://user-images.githubusercontent.com/74269598/212283964-7e087dde-18b9-4173-b009-b1ac14fc00ac.png)

**Comment by martabal** [comment]
Ok  I had some issues with my github actions, please change repository from `martabal/immich:latest` to `martabal/immich:v1.41.1_64-dev`.

**Comment by blaine07** [comment]
> Ok I had some issues with my github actions, please change repository from `martabal/immich:latest` to `martabal/immich:v1.41.1_64-dev`

Do I still keep the machine learning disabled env? Will that repository work fine on my Unraid Server(it won’t matter it’s on Unraid)?

**Comment by blaine07** [comment]
![Screen Shot 2023-01-13 at 6 42 07 AM](https://user-images.githubusercontent.com/63940698/212322861-f4faee34-b1a2-4462-bc0a-5512272a6fce.png)
Well I see this in the logs; it isn't spamming log? No idea what it's actually doing, but it ISN"T spamming log :-)

**Comment by martabal** [comment]
> Do I still keep the machine learning disabled env? Will that repository work fine on my Unraid Server(it won’t matter it’s on Unraid)?

Yeah just edit the tag just to test, I am resolving the issue with the `latest` tag, (it will be solved in a few minutes, then you'll can restore the `latest` tag).

> Well I see this in the logs; it isn't spamming log? No idea what it's actually doing, but it ISN"T spamming log :-)

This is "normal" logs. Immich generates a lot of log during startup. There is a [PR](https://github.com/immich-app/immich/pull/1318) on Immich to add multiple log levels, it will be much easier to deal with logs

**Comment by martabal** [comment]
@blaine07 Problem solved, you can restore the `latest` tag :)

**Comment by blaine07** [comment]
> @blaine07 Problem solved, you can restore the `latest` tag :)

Appears to be fine again moving back over the Latest.

THANK YOU!
